care/facility/api/serializers/patient.py

```diff
@@ -465,10 +470,15 @@ def validate_empty_values(self, data):
     class Meta:
         model = PatientNotes
         fields = (
+            "id",
             "note",
             "facility",
             "created_by_object",
             "created_by_local_user",
-            "created_date",
+            "edit_window_seconds",
```

Comment:
its not required to add this field to both fields and read_only_fields

Reply:
Adding it to only read_only_fields throws an exception : `AssertionError: The field 'edit_window_seconds' was declared on serializer PatientNotesSerializer, but has not been included in the 'fields' option.`. You mean to remove it from `read_only_fields` and keep it only in fields?